web/src/pages/billing/SequencingGroupsByMonth.tsx

```diff
@@ -0,0 +1,263 @@
+import { debounce } from 'lodash'
+import * as React from 'react'
+import { useRef } from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
+import { useMeasure } from 'react-use'
+import { Button, Card, Grid, Message } from 'semantic-ui-react'
+import LoadingDucks from '../../shared/components/LoadingDucks/LoadingDucks'
+import ProjectSelector, { IMetamistProject } from '../project/ProjectSelector'
+
+import * as Plot from '@observablehq/plot'
+import { PaddedPage } from '../../shared/components/Layout/PaddedPage'
+import { SequencingGroupApi } from '../../sm-api'
+import './components/BillingCostByTimeTable.css'
+
+type ProjectHistory = {
+    [key: string]: {
+        [key: string]: number
+    }
+}
+
+interface TypeData {
+    date: Date
+    type: string
+    count: number
+}
+
+/* eslint-disable @typescript-eslint/no-explicit-any  -- too many anys in the file to fix right now but would be good to sort out when we can */
```

Comment:
This comment was added to another page that was created before we had proper linting. New pages do need to conform to slightly higher standards so it would be good to remove this and fix the uses of explicit any.